keps/NNNN-kep-template/kep.yaml

```diff
@@ -31,6 +31,19 @@ stage: alpha|beta|stable
 # worked on.
 latest-milestone: "v1.19"
 
+
```

Comment:
@justaugustus @LappleApple 

I think this might be a good point to update the metadata so we know velocity of API and features.
Might be a better indicator of what's coming next in a release and ensure we have testing in place.... and probably a lot of other good could come of it.